components/profile_picture.jsx

```diff
@@ -26,6 +26,7 @@ export default class ProfilePicture extends React.PureComponent {
         isBusy: PropTypes.bool,
         isRHS: PropTypes.bool,
         hasMention: PropTypes.bool,
+        helperClass: PropTypes.string,
```

Comment:
Probably I would use `className` here because is what it is, the class name for the `ProfilePicture`. Actually, I think that should be at top level. `helperClass` have no meaning at all from the outside perspective.

Reply:
Changing to `wrapperClass` because this is slightly modifying the styling of the existing class for this specific instance.